lib/schema_migrator.rb

```diff
@@ -10,7 +10,7 @@ def initialize(index_name, cluster: Clusters.default_cluster, wait_between_task_
 
   def reindex
     index_group.current.with_lock do
-      response = Services.elasticsearch(cluster: cluster, timeout: 60).reindex(
+      response = Services.elasticsearch(hosts: 'http://localhost:9200?slices=auto', cluster: cluster, timeout: 60).reindex(
```

Comment:
`Services.elasticsearch` takes `hosts` or `cluster`, but only uses one of them. If hosts is present it will ignore cluster:

```
    Elasticsearch::Client.new(
      hosts: cluster ? cluster.uri : hosts,
```

In this case, you'll need to add `slices=auto` to the `cluster.uri` (or find some other way to make this config change).

Reply:
Actual documentation is pending, when I can refine it ;)

I'm going to take Michael's option, as it does what you want nice and simply.